config/prow/cluster/monitoring/mixins/grafana_dashboards/boskos.jsonnet

```diff
@@ -39,7 +39,7 @@ dashboard.new(
         {type: "gke-project", friendly: "GKE project"},
         {type: "gpu-project", friendly: "GPU project"},
         {type: "ingress-project", friendly: "Ingress project"},
-        {type: "istio-project", friendly: "Istio project"},
```

Comment:
istio-project pool is unused at this point thanks to https://github.com/kubernetes/test-infra/pull/17213